build.xml

```diff
@@ -1176,6 +1176,24 @@ osname=macosx;processor=x86;processor=x86-64;processor=ppc
     </subant>
   </target>
 
+  <target name="deploy-local" depends="dist" description="deploy snapshot version to local repository">
```

Comment:
I'd suggest target name "install" (matches maven goal "install:install", which installs artifacts into the local repository, the installation into a remote repository is "deploy:deploy").

And for the description: "Install jna and jna-platform artifacts into local maven repository". There is nothing preventing the user to run install on a non-snapshot build.